backend/app/Models/Transaction.php

```diff
@@ -9,6 +9,10 @@ class Transaction extends Model
 {
     use HasFactory;
 
+    protected $casts = [
+        'id' => 'string',
+    ];
+
```

Comment:
as i found out later, this is not enough to handle the model's id being uuid

refer to #13

1. copy trait file to this PR as well  `backend/app/Traits/Uuid.php`
2. do the same as on pr13, `backend/app/Models/User.php:13` (right side of diff)
3. remove this `protected $casts`
4. and line same line from pr13 as well `backend/app/Models/User.php:41-55` (right side of diff)